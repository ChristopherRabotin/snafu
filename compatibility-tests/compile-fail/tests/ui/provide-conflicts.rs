use snafu::prelude::*;

#[derive(Debug, Snafu)]
#[snafu(provide(chain, u8 => 0))]
struct Error;

fn main() {}
