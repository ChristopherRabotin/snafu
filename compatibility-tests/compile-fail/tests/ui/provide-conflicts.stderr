error: May only chain to references; please add `ref` flag
 --> tests/ui/provide-conflicts.rs:4:9
  |
4 | #[snafu(provide(chain, u8 => 0))]
  |         ^^^^^^^^^^^^^^^^^^^^^^^
